onadata/libs/utils/common_tools.py

```diff
@@ -104,3 +104,28 @@ def get_response_content(response):
         contents = response.content
 
     return contents
+
+
+def json_stream(data, json_string):
+    """
+    Generator function to stream JSON data
```

Comment:
let's add a definition of the arguments, including that `data` must be an iterator